prisma/schema.prisma

```diff
@@ -0,0 +1,66 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
```

Comment:
Do we really need these first two lines? 

I understand that Prisma would have a `.prisma` file to start with, but I'm not sure if these lines are very informative. Prisma is a required package in `package.json`, in my opinion, seeing prisma as a dependency should hint to the dev that there are prima-specific files like this one.

Reply:
they were auto-generated by prisma. I see no need to keep them around